chatmaild/src/chatmaild/filtermail.py

```diff
@@ -312,7 +312,7 @@ def sync_handle_DATA(self, envelope):
         client = SMTPClient(
             "localhost",
             self.config.postfix_reinject_port_incoming,
-            source_address=("127.0.0.2", 0),
+            source_address=("127.0.0.1", 0),
```

Comment:
See comment above, the reason for using 127.0.0.2 is to make opendkim verify DKIM signature. For messages that come from 127.0.0.1 it signs the message instead.